pd-control.md

```diff
@@ -597,22 +642,27 @@ Usage:
 }
 ```
 
-### `scheduler [show | add | remove]`
+### `scheduler [show | add | remove | pause | resume | config]`
 
 Use this command to view and control the scheduling strategy.
 
 Usage:
 
 ```bash
->> scheduler show                             // Display all schedulers
->> scheduler add grant-leader-scheduler 1     // Schedule all the leaders of the regions on store 1 to store 1
->> scheduler add evict-leader-scheduler 1     // Move all the region leaders on store 1 out
->> scheduler add shuffle-leader-scheduler     // Randomly exchange the leader on different stores
->> scheduler add shuffle-region-scheduler     // Randomly scheduling the regions on different stores
->> scheduler remove grant-leader-scheduler-1  // Remove the corresponding scheduler
+>> scheduler show                                 // Display all schedulers
+>> scheduler add grant-leader-scheduler 1         // Schedule all the leaders of the regions on store 1 to store 1
+>> scheduler add evict-leader-scheduler 1         // Move all the region leaders on store 1 out
+>> scheduler add shuffle-leader-scheduler         // Randomly exchange the leader on different stores
+>> scheduler add shuffle-region-scheduler         // Randomly scheduling the regions on different stores
+>> scheduler remove grant-leader-scheduler-1      // Remove the corresponding scheduler
+>> schedule pause balance-region-scheduler 10     // Pause the balance-region scheduler for 10 seconds
```

Comment:
should be scheduler?

Reply:
See https://github.com/pingcap/docs-cn/pull/3305/files#diff-71fa457167c89bb4e3cb651dc0484326R951. And the latest Chinese doc uses `schedule`. Is it a typo? @rleungx

Reply:
Confirmed it's a typo from the Chinese doc. Fixed in this PR and https://github.com/pingcap/docs-cn/pull/3995.